code/__DEFINES/atmospherics.dm

```diff
@@ -3,7 +3,6 @@
 #define GAS_PL	(1 << 2)
 #define GAS_CO2	(1 << 3)
 #define GAS_N2O	(1 << 4)
-#define GAS_A_B	(1 << 5)
```

Comment:
I'd leave this define in for the sake of consistency - In case someone want to use it for another purpose. 

It is weird to have a define list of all gas only to have one type explicitly missing

Reply:
Done